web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -78,8 +78,12 @@ describe('HFI Calculator Page', () => {
       cy.intercept('GET', 'api/hfi-calc/fire-centres', {
         fixture: 'hfi-calc/fire_centres.json'
       }).as('getFireCentres')
+      cy.intercept('GET', 'api/hfi-calc/fuel_types', {
```

Comment:
I think we should throw this inside interceptLoad - we can pass interceptLoad two json paths. Sonarcloud is legitimately picking this up as duplicate code.